modules/ROOT/pages/partner-manager-create-message-type.adoc

```diff
@@ -60,37 +59,38 @@ Only segment level customization is supported.
 [[target-at-host]]
 == Target at Host
 
-These are application message structure (JSON or XML) to which inbound EDI transactions are translated into and sent to your back-end applications.
+The application message structure (JSON or XML) to which inbound EDI transactions are translated and sent to your back-end applications.
 
 [%header%autowidth.spread]
 |===
```

Comment:
ditto re. standardizing to all fragments, without periods (or all sentences, with periods) if possible
GLOBAL for Description columns in tables

Reply:
Done